jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-typedarray-prototype.c

```diff
@@ -232,6 +238,13 @@ ecma_builtin_typedarray_prototype_map (ecma_value_t this_arg, /**< this object *
     ecma_free_value (current_index);
     ecma_free_value (element);
 
+    if (ecma_arraybuffer_is_detached (src_info_p->array_buffer_p))
```

Comment:
There are several check like this, which sould be after `if (ECMA_IS_VALUE_ERROR (mapped_value))`. Please check it in the whole PR.